### ISSUE TITLE
Front end bug genesis celestia website

### ISSUE THREAD

**Issue by DikaCream** [celestiaorg/celestia-app#2718]
### Summary of Bug

Console log result : [DEPRECATED] Use `createWithEqualityFn` instead of `create` or use `useStoreWithEqualityFn` instead of `useStore`. They can be imported from 'zustand/traditional'. 

https://github.com/pmndrs/zustand/discussions/1937
[DEPRECATED] Passing a vanilla store will be unsupported in a future version. Instead use `import { useStore } from 'zustand'`.

### Version

Latest

### Steps to Reproduce

https://github.com/pmndrs/zustand/discussions/1937

### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by DikaCream** [comment]
Additional info : Genesis website has been  blocked by CORS policy 

Access to XMLHttpRequest at 'https://genesis-api.celestia.org/api/v1/airdrop/eligibility' from origin 'https://genesis.celestia.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
rainbowkit.js:97 TypeError: Cannot read properties of undefined (reading 'status')
    at rainbowkit.js:74:58
genesis-api.celestia…rdrop/eligibility:1
 Failed to load resource: net::ERR_FAILED
DevTools failed to load source map: Could not load

There is 16 deprecated logs + 2 error logs

**Comment by rootulp** [comment]
@DikaCream did you observe this error on https://genesis.celestia.org/ ? If so, this issue should probably be on a different GitHub repo. Since the Celestia airdrop claim period has ended, closing as won't fix for now. Please re-open with more complete **Steps to Reproduce** if I'm misunderstanding something.

**Comment by DikaCream** [comment]
> more

Can you tag me the repo on which repo I should report about front end bug celestia Genesis website ?  For some people Website isn't response after sign metamask signature

**Comment by rootulp** [comment]
Thanks for originally reporting but since the claim period is over I don't think you need to recreate this issue on a different repo b/c it likely won't be addressed.
